app/models/order_balance.rb

```diff
@@ -17,11 +19,13 @@ def to_f
     if customer_balance_enabled?
       order.new_outstanding_balance
     else
-      order.outstanding_balance
```

Comment:
I think I'd rather see this `#to_f` method renamed to `#amount`, and the current `#amount` method being renamed to something like `#display_amount` (which is consistent with all the other places in the codebase where we show something as a Spree::Money object), but it's just personal preference.

Reply:
I'll consider it when we remove the `old_oustanding_balance` branch, if you don't mind.